CHANGELOG.md

```diff
@@ -33,6 +33,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - 💥 Removed the `SyntaxHighlighter` component. This is now built into `Markdown` [#562](https://github.com/plotly/dash-core-components/pull/562).
 - Removed the `containerProps` prop in `Markdown` - after the refactor of [#562](https://github.com/plotly/dash-core-components/pull/562), its function is served by the `id`, `className`, and `style` props. [#569](https://github.com/plotly/dash-core-components/pull/569)
 
+### Updated
+- Upgraded plotly.js to 1.48.3 [#571](https://github.com/plotly/dash-core-components/pull/571)
+  - [Feature release 1.48.0](https://github.com/plotly/plotly.js/releases/tag/v1.48.0) which contains:
+    -
+    -
+    -
```

Comment:
@alexcjohnson Will need your awesome Plotly.js knowledge to fill in this part with the more relevant features.

Reply:
features summarized in https://github.com/plotly/dash-core-components/pull/571/commits/18d4d9f4fd85ee61b8289b177fbb8283dcd81a5a